.github/workflows/e2e-testnet.yml

```diff
@@ -0,0 +1,45 @@
+name: E2E tests / Testnet
+
+on:
+  schedule:
+    - cron: '*/20 * * * *'
+  workflow_dispatch:
+  pull_request: # TODO: remove this trigger before merging to main
+
+jobs:
+  e2e-testnet-tests:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./e2e
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "14.x"
+
+      # TODO: Temprorarily we use fixed version of tbtc.js. Ultimately, we would
+      # need to point to the latest deployed version.
+      - run: npm install --save-exact @keep-network/tbtc.js@0.19.4-ropsten.2
```

Comment:
Do we have a task in the backlog to cover that?

Reply:
Added, https://keepnetwork.atlassian.net/browse/CP-56.